web/src/games/kriegspiel/Board.tsx

```diff
@@ -0,0 +1,770 @@
+                }}
+              >
+                <rect
+                  width="0.9"
+                  height="0.9"
+                  x="0.05"
+                  y="0.05"
+                  fill={fictionColor(editFiction)}
+                  stroke={id === editState ? pico8Palette.red : pico8Palette.dark_grey}
+                  strokeWidth={id === editState ? 0.15 : 0.05}
+                />
+                {renderStr(Game.objDataList[type].objRender)}
+              </g>
+            );
+          }, objTypeList)}
+          {gTranslate(
+            renderLayer((type, oid) => {
+              const id = oid + 6;
+              return (
+                <g
+                  onClick={() => {
+                    editorClick(id);
+                  }}
+                >
+                  <rect
+                    width="0.9"
+                    height="0.9"
+                    x="0.05"
+                    y="0.05"
+                    fill={fictionColor(editFiction)}
+                    stroke={id === editState ? pico8Palette.red : pico8Palette.dark_grey}
+                    strokeWidth={id === editState ? 0.15 : 0.05}
+                  />
+                  {renderStr(Game.renderPlaceByType(type)[0])}
+                </g>
+              );
+            }, strongholdTypeList),
+            0,
+            1,
+          )}
+        </svg>
+        <input
+          type="button"
+          value="Change Color"
+          onClick={() => {
+            setEditFiction(dualPlayerID(editFiction));
+          }}
+        />
+        <input type="button" value="Reset Board" onClick={() => moves.load(Game.onlyMap)} />
+      </div>
+      {/* Game Data */}
+      <form>
+        <label>GameData:</label>
+        <textarea
+          name="gameData"
+          id="gameData"
+          //cols={20}
+          rows={10}
+          style={{ width: '90%', height: '60%', resize: 'vertical' }}
+          value={gameData}
+          onChange={(e) => setGameData(e.target.value)}
+        ></textarea>
+
+        <input type="button" value="Export Game" onClick={() => setGameData(exportGame(G))} />
+        <input type="button" value="Load Game" onClick={() => moves.load(gameData)} />
+        <input type="button" value="Merge Game" onClick={() => moves.merge(gameData)} />
+        <input
+          type="button"
+          value="Copy Data"
+          onClick={() => {
+            // Get the text field
+            const copyText = document.getElementById('gameData') as HTMLInputElement;
+            // Select the text field
+            copyText.select();
+            copyText.setSelectionRange(0, 99999); // For mobile devices
+            // Copy the text inside the text field
+            navigator.clipboard.writeText(gameData);
+          }}
+        />
+        <input type="button" value="Remove Data" onClick={() => setGameData('')} />
+      </form>
+    </div>
+  );
+  // render all
+  return (
+    <main>
```

Comment:
Please make sure you wrap your board with <GameLayout, like here:
https://github.com/freeboardgames/FreeBoardGames.org/blob/4361c9f25e431b91ce9ff99cba78855674b00046/web/src/games/tictactoe/board.tsx#L93

and that you pass gameOver when the game is over. This will allow users to navigate back to the main menu and replay your game (or switch to another game) when it is over.